docs/task.md

```diff
@@ -80,7 +80,7 @@ but yet without parameter values such as how, how many, what, where from, where
     },
     "@context": [
         "https://smartdatamodels.org/context.jsonld",
-        "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfcontext.jsonld"
+        "https://raw.githubusercontent.com/shop4cf/data-models/master/docs/shop4cfcontext.jsonld"
```

Comment:
Make sure that the link [https://raw.githubusercontent.com/shop4cf/data-models/master/docs/shop4cfdefinitions.jsonld](https://raw.githubusercontent.com/shop4cf/data-models/master/docs/shop4cfdefinitions.jsonld) is reachable after the push, for now it is not working

Reply:
It becomes reachable once we merge this PR. Do you think we can do this now?

That left open issue (defining `status`) can be added later (soon).